module/scratchpad.lua

```diff
@@ -163,17 +163,28 @@ end
 --- Turns the scratchpad off if it is focused otherwise it raises the scratchpad
 function Scratchpad:toggle()
     local is_turn_off = false
+    local matches = self:find()
     if self.dont_focus_before_close then
-        local matches = self:find()
-        if matches[1] and matches[1].first_tag then
-            is_turn_off = matches[1].first_tag.selected
+        if matches[1] then
+            local current_tag = matches[1].screen.selected_tag
+            for k, tag in pairs(matches[1]:tags()) do
+                if tag == current_tag then
+                    is_turn_off = true
+                    break
+                else
+                    is_turn_off = false
```

Comment:
The is_turn_off is used below to determine if to turn_on/off

Reply:
Sure but it is already set to false at the start right? So in the loop the variable will either be set to true and then break the loop or it will be set to false again which is what I mean with the redundant else block. Or am I missing something?

Doesn't really matter in the end anyway but it might be easier to read that way

Reply:
@Nooo37 Yeah, didn't notice that